tests/EnumAccessorTest.php

```diff
@@ -92,4 +135,20 @@ public function it_doesnt_break_related_properties()
         $this->assertInstanceOf(Client::class, $order->client);
         $this->assertEquals($client->id, $order->client->id);
     }
+    
+    private function getEnumVersion()
+    {
+        $raw_version = \PackageVersions\Versions::getVersion('konekt/enum');
+        
+        $parts = explode('@', $raw_version);
+        
+        return $parts[0];
+    }
+    
+    private function getEnumVersionMayor()
```

Comment:
There's a typo, I guess you wanted "Major" not "Mayor" ;)

Reply:
fixed